### ISSUE TITLE
For the same dataset, training loss is higher than validation loss  

### ISSUE THREAD

**Issue by mohammadAbdolhosseiniMoghaddam** [keras-team/keras#10998]
I have faced a weird problem. My training loss for a a specific data set is always higher than my validation set. It does not matter how many dataset I want to train. For example, I tested it just for 2 training data set and 3 iteration and I expected that it over fits but strangely it did not happen. I also selected the same validation and training set but the loss value  was different. any help is appreciated. I would like to mention that I am  training on time series data and I am using lstm layers.

```
        model = Sequential()
        model.add(LSTM(60, input_shape=(train_X.shape[1], train_X.shape[2]),return_sequences=False))
        model.add(Dense(1,activation='sigmoid'))
        sgd = optimizers.SGD(lr=0.01, decay=1e-2, momentum=0.9)
        model.compile(loss='mae', optimizer='sgd')

train_X.shape --->(1, 5, 32)
  history = model.fit(train_X, scaled_train_y, epochs=3, batch_size=2, validation_data=(train_X, scaled_train_y), verbose=2, shuffle=True)

Train on 1 samples, validate on 1 samples
Epoch 1/3
 - 14s - loss: 0.3944 - val_loss: 0.3891
Epoch 2/3
 - 0s - loss: 0.3891 - val_loss: 0.3839
Epoch 3/3
 - 0s - loss: 0.3839 - val_loss: 0.3786

**Comment by tRosenflanz** [comment]
Train loss gets better during the epoch so by the end training loss will be lower than at the start of the epoch.
